README.rst

```diff
@@ -29,6 +29,7 @@ Currently tested only for
 
 -   Django 2.2 under Python 3.6-3.7
 -   Django 3.0 under Python 3.6-3.8
```

Comment:
FWIW, it seems like [2.2](https://docs.djangoproject.com/en/3.1/releases/2.2/) and [3.0](https://docs.djangoproject.com/en/3.1/releases/3.0/) also supports 3.8 and 3.9.

Reply:
👍